webapp/components/ContributionForm/index.vue

```diff
@@ -137,7 +138,10 @@ export default {
       this.$refs.contributionForm.update('content', value)
     },
     returnLocaleName(locale) {
-      if (this.$i18n.locale() === locale.code) {
+      if (
+        (this.contribution && this.contribution.language === locale.code) ||
+        (!this.contribution && this.$i18n.locale() === locale.code)
+      ) {
```

Comment:
What happens in the else case?

Reply:
good question @Tirokk, I didn't think there was a case where `this.$i8n.locale()` is undefined :thinking: 
what would you expect to happen if there is no locale?

Reply:
Just use english as the world language @mattwr18 , I think. Or?
If the else case is not possible, we don’t need an ìf` here?!

Reply:
Addressed in our latest pairing session @Tirokk... unfortunately, I forgot to add you as a co-author, but I think you know I appreciate you!!